src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+
+  if (days >= 3 && days < 7) {
+    return rentCost * days - 20;
+  }
```

Comment:
Consider removing the 'else' clause after an 'if' statement that contains a 'return'. This simplifies the control flow and makes the code easier to read.

Reply:
The same here.